plugins/processors/t128_rate/README.md

```diff
@@ -0,0 +1,36 @@
+# T128 Rate Processor Plugin
+
+The `t128_rate` processor computes the point to point difference of a field.
+
+### Configuration:
+
+```toml
+[[processors.t128_rate]]
+  ## If more than this amount of time passes between data points, the
+  ## previous value will be considered old and the rate will be recalculated
+  ## as if it hadn't been seen before. A zero expiration means never expire.
+  # expiration = "0s"
+
+  ## For the fields who's key/value pairs don't match, should the original
+  ## field be removed?
+  # remove-original = true
+
+[processors.t128_rate.fields]
+  ## Replace fields with their rates, renaming them if indicated
+  # "/rate/metric" = "/total/metric"
+  # "/inline/replace" = "/inline/replace"]
```

Comment:
Should there be an opening bracket to match the closing bracket at the end of this line?

Reply:
Good catch. There should be no closing bracket.